react-app/src/components/PortfolioScreen/StakesPanel.tsx

```diff
@@ -0,0 +1,142 @@
+          <tr>
+            <ColumnSortContext.Provider value={columnSortContextValue}>
+              <ColumnHeader
+                id="name"
+                titleId="StakesPanel.name"
+                className="uppercase bg-gray-50 border-b-[1px] border-gray-200"
+              />
+              <ColumnHeader
+                id="staked"
+                titleId="StakesPanel.staked"
+                className="uppercase bg-gray-50 border-b-[1px] border-gray-200"
+              />
+              <ColumnHeader
+                id="rewards"
+                titleId="StakesPanel.rewards"
+                className="hidden uppercase bg-gray-50 border-b-[1px] border-gray-200 sm:table-cell"
+              />
+              <ColumnHeader
+                id="expectedReturns"
+                titleId="StakesPanel.expectedReturns"
+                className="hidden uppercase bg-gray-50 border-b-[1px] border-gray-200 sm:table-cell"
+              />
+              <ColumnHeader
+                id="votingPower"
+                titleId="StakesPanel.votingPower"
+                className="hidden uppercase bg-gray-50 border-b-[1px] border-gray-200 sm:table-cell"
+              />
+            </ColumnSortContext.Provider>
+          </tr>
+        </thead>
+
+        <tbody>
+          {stakes.map((stake) => (
+            <tr
+              key={stake.delegation.validatorAddress}
+              className="text-sm font-normal leading-5 text-gray-500"
+            >
+              <td className="flex items-center py-4 px-6">
+                {/* TODO: Fetch from Graphql API */}
+                <div
+                  className={cn("w-9", "h-9", "rounded-full", "bg-blue-700")}
+                />
+                <div className="ml-3">
+                  <h3 className="text-sm font-medium text-likecoin-green">
+                    {stake.validator.moniker}
+                  </h3>
+                  <p
+                    className={cn(
+                      "text-xs",
+                      "font-medium",
+                      "leading-[14px]",
+                      "text-likecoin-vote-color-yes"
+                    )}
+                  >
+                    <LocalizedText messageID="StakesPanel.active" />
+                  </p>
+                </div>
+              </td>
+              <td className="px-6">
+                {convertBigNumberToMillifiedIntegerString(stake.balance.amount)}
+              </td>
+              <td className="hidden px-6 sm:table-cell">
+                {convertBigNumberToFixedPointString(stake.reward.amount, 9)}
+              </td>
+              <td className="hidden px-6 sm:table-cell">
+                {/* TODO: Fetch from Graphql API */}
+                15.76%
+              </td>
+              <td className="hidden px-6 sm:table-cell">
+                {/* TODO: Fetch from Graphql API */}
+                3.69%
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
```

Comment:
Make sure you either create a generic Table component or modify SectionedTable to suit your needs